src/algorithms/channel/adapters/channel.h

```diff
@@ -86,6 +86,8 @@ class Channel : public ChannelInterface
     inline std::shared_ptr<TrackingInterface> tracking() const { return trk_; }
     inline std::shared_ptr<TelemetryDecoderInterface> telemetry() const { return nav_; }
 
+    inline uint32_t fsm_state() { return channel_fsm_->state(); }
```

Comment:
Please mark this member function as `const`, since it does not (and it should not) modify any internal member. So it would be:
`inline uint32_t fsm_state() const { return channel_fsm_->state(); }`

Reply:
Done in commit https://github.com/gnss-sdr/gnss-sdr/pull/419/commits/585754a665345c9629dd327d1d81dfc568cdf722

Reply:
Fixed in commit https://github.com/gnss-sdr/gnss-sdr/pull/419/commits/585754a665345c9629dd327d1d81dfc568cdf722